core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -367,8 +372,8 @@ public AlluxioURI getUri() {
     /**
      * @return the {@link UnderFileSystem} instance
      */
-    public UnderFileSystem getUfs() {
-      return mUfs;
+    public UfsManager.UfsClient getUfsClient() {
+      return mUfsClient;
```

Comment:
Is this `UfsClient` well formed? If not we could consider returning `Resource<UnderFileSystem>` instead.